code/modules/food_and_drinks/recipes/tablecraft/recipes_pastry.dm

```diff
@@ -399,6 +399,67 @@
 	result = /obj/item/food/donkpocket/gondola
 	subcategory = CAT_PASTRY
 
+/datum/crafting_recipe/food/donkpocket/donkmousebrown
+	time = 15
+	name = "Brown Mouse-pocket"
+	reqs = list(
+		/obj/item/food/pastrybase = 1,
+		/obj/item/food/deadmouse = 1
+	)
+	result = /obj/item/food/donkpocket/donkmousebrown
+	subcategory = CAT_PASTRY
+
+/datum/crafting_recipe/food/donkpocket/donkmousebrown/check_requirements(mob/user, list/collected_requirements) //Important override so that a specific icon_state is required for the specific mouse pocket to be crafted. (So you cant make brown mouse pockets with white mice).
+	var/obj/item/food/deadmouse = collected_requirements[/obj/item/food/deadmouse][1]
+	var/mouse_state = deadmouse.icon_state //Getting the icon_state of the deadmouse and setting it equal to mouse_state.
+	if(mouse_state == "mouse_brown_dead")
```

Comment:
When is this not true? Admin intervention?

I think either way, I'd rather this check `initial(deadmouse.icon_state)` (after correctly checking `var/obj/item/food/deadmouse` to `var/obj/item/food/deadmouse/deadmouse`).